src/jit/lowerxarch.cpp

```diff
@@ -905,7 +905,28 @@ void Lowering::TreeNodeInfoInit(GenTree* tree)
         }
     }
 
+    TreeNodeInfoInitCheckByteable(tree);
+
+    // We need to be sure that we've set info->srcCount and info->dstCount appropriately
+    assert((info->dstCount < 2) || (tree->IsMultiRegCall() && info->dstCount == MAX_RET_REG_COUNT));
+}
+
+//------------------------------------------------------------------------
+// TreeNodeInfoInitCheckByteable: Check the tree to see if "byte-able" registers need to be
+// required, and set the tree node info accordingly.
```

Comment:
Minor: this comment could be simplified as follows:

'if "byte-able" registers are required,'

instead of 'need to be required'